File: CS/WindowsApplication12/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace WindowsApplication12
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            for (int i = 0; i < 5; i++)
                dataTable1.Rows.Add(new object[] { i, "Product" + i});
            for (int i = 0; i < 30; i++)
                dataTable2.Rows.Add(new object[] { i, i%5, "details"});
        }
    }
}

File: VB/WindowsApplication12/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace WindowsApplication12
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			For i As Integer = 0 To 4
				dataTable1.Rows.Add(New Object() { i, "Product" & i})
			Next i
			For i As Integer = 0 To 29
				dataTable2.Rows.Add(New Object() { i, i Mod 5, "details"})
			Next i
		End Sub
	End Class
End Namespace